layouts/shortcodes/eol-releases.html

```diff
@@ -0,0 +1,34 @@
+<h2 id="non-active-branch-history">Non-Active Branch history</h2>
+<p>
+    These releases are no longer supported.
+</p>
+<div>
+    <table class="table table-striped col-md-8">
+        <thead class="thead">
+            <tr>
+                <th class="col-md-2">Minor Version</th>
+                <th class="col-md-2">Final Patch Release</th>
+                <th class="col-md-2">EOL Date</th>
+                <th class="col-md-4">Note</th>
+            </tr>
+        </thead>
+        <tbody>
+            {{ range $eolRelease := .Site.Data.releases.eol.branches }}
+            <tr>
+                <td>
+                    {{ $eolRelease.release }}
+                </td>
+                <td>
+                    {{ $eolRelease.finalPatchRelease }}
+                </td>
+                <td>
+                    {{ $eolRelease.endOfLifeDate }}
+                </td>
+                <td>
+                    {{ $eolRelease.note }}
```

Comment:
Can we support Markdown within notes? I'd prefer to.

Try https://gohugo.io/functions/markdownify/ for this.

Reply:
I tried using Markdownify. However, they weren't working when tried the following:
```yaml
  notes: |-
    This is a note
```

Doing the following as suggested in your other comment solved this:
```yaml
  notes: >-
    This is a note
```